frontend/src/metabase/entities/fields.js

```diff
@@ -81,10 +81,14 @@ const Fields = createEntity({
       ),
       withNormalize(FieldSchema),
     )(({ id: fieldId }) => async (dispatch, getState) => {
-      const { field_id: id, values } = await MetabaseApi.field_values({
+      const {
+        field_id: id,
+        values,
+        has_more_values = false,
```

Comment:
this field is a not-null field in BE, so maybe we don't need to have a default value for it?